### PR TITLE
Some minor corrections / improvements, listed by chapter

### PR DESCRIPTION
**Pull request by zasnicoff**

### CODE REVIEW

**Review by zasnicoff — COMMENTED** [review]
The first commit has been included in a previous pull request, sorry for the duplicate.

**Comment by aantonop** [comment]
We're going to need to cherry-pick this pull request, as it combines a lot of stuff from a lot of different sections. Too much for one pull request....

**Comment by zasnicoff** [comment]
I need to learn more about GitHub and best practices. Had the impression that the owner could approve/reject/edit commits individually. It seems to be a lot faster/easier for me to make commits to the same branch/pull request. Sorry about that.

**Comment by aantonop** [comment]
No worries! A great way to learn is to practice with a real pull request.
How about trying the following: create a new branch on your fork and merge
a few of the commits at a time into each new branch, making a PR for each,
splitting all this work into different PRs?

Basically repackage these same commits into multiple branches and pull
requests.

Or, I can do that myself some time tomorrow, cherry picking the commits I
want to merge.

Thanks for all this work!

File: consensus.asciidoc [hunk 1]
```diff
@@ -4,8 +4,12 @@
 
 When it comes to decentralized record keeping, which is how blockchain technology is often described, it becomes infeasible to rely on trust alone to ensure that factual information is added to the record books. This problem arises in decentralized networks as a result of the lack of a central entity that would typically decide what should and should not be considered as fact. It's difficult to avoid this issue when working with blockchain technology because it is this lack of a central entity that is one of the main contributors to blockchain's popularity. In order to circumvent this issue a network must have a system in place by which a consensus can be reached by the nodes in the network.
 
+//Philosofical: In effect, you are trusting the system. I never appreciated the idea of a "trustless" system, instead one might look at it as shifting the trust in someone to trust in the system.
```
Comment (fjrojasgarcia) [inline consensus.asciidoc:7]: I like so much this rethink about _"trustless system"_. I do not know if my opinion will have to do with the fact that English is not my native language. 

I also agree with the considerations that Andreas makes in some of his talks, about which little effort is made to find **terms better adapted** to innovations and how there are tends to reuse old terms that do not faithfully reflect the underlying concept.

File: consensus.asciidoc [hunk 2]
```diff
@@ -4,8 +4,12 @@
...
+
 A consensus protocol is a system that allows network nodes to reach an agreement on what should be considered as truth. When referring to a blockchain network this protocol is used to determine which copy of the blockchain is the true copy. This is required in order to prevent malicious parties from frauding the network and to guarantee the integrity of the blockchain is maintained. There are different types of consensus protocols in use in blockchain technologies, we will focus on proof of work (*POW*) and proof of stake (*POS*).
 
+//I am probably not suited to contribute and write about Ethereum consensus, but as a reader I would like to read more about how consensus work to provide immutability, before entering into POW/POS and specifics. I understand this is a work in progress.
```
Comment (fjrojasgarcia) [inline consensus.asciidoc:11]: Just a little note about such a trancendental point. For me, through the **"Vlad Zamfir's"** works, it was fascinating to discover that there is a branch of engineering specialized in Consensus, something that surely those deeply interested in the subject should study and cultivate.

Comment (zasnicoff) [reply consensus.asciidoc:11]: Will definitely look into it, thanks.

**Comment by zasnicoff** [comment]
Alright, did my homework, little bit more comfortable with GitHub at this point.
Thank you @aantonop  for the suggestions, thank you @fjrojasgarcia for all the feedbacks.
I did repack my commits and will PR them now. Tried to combine them in additional-text; corrections; suggestions and formatting. Hopefully easier to evaluate and merge.
I guess the original #409 PR could be rejected / deleted.

**Pull request closed without merging**